Slack.Automation/Promact.Erp.Core/Controllers/OAuthController.cs

```diff
@@ -52,7 +64,7 @@ public IHttpActionResult RefreshToken(string refreshToken)
             }
             catch (Exception ex)
             {
-                var errorMessage = string.Format("{0}. Error -> {1}", StringConstant.LoggerErrorMessageOAuthControllerRefreshToken, ex.ToString());
+                var errorMessage = string.Format("{0}. Error -> {1}", _stringConstant.LoggerErrorMessageOAuthControllerRefreshToken, ex.ToString());
```

Comment:
Why have you used so many try catch in Refresh token method? You are catching a generic exception which we don't do.